athena-jdbc/src/test/java/com/amazonaws/connectors/athena/jdbc/integ/RedshiftIntegTest.java

```diff
@@ -0,0 +1,281 @@
+/*-
+ * #%L
+ * athena-jdbc
+ * %%
+ * Copyright (C) 2019 - 2020 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.connectors.athena.jdbc.integ;
+
+import com.amazonaws.athena.connector.integ.data.ConnectorVpcAttributes;
+import com.amazonaws.athena.connector.integ.stacks.ConnectorWithVpcStack;
+import com.amazonaws.athena.connector.integ.IntegrationTestBase;
+import com.amazonaws.services.athena.model.Row;
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+import software.amazon.awscdk.core.RemovalPolicy;
+import software.amazon.awscdk.core.SecretValue;
+import software.amazon.awscdk.core.Stack;
+import software.amazon.awscdk.services.ec2.SecurityGroup;
+import software.amazon.awscdk.services.ec2.Vpc;
+import software.amazon.awscdk.services.ec2.VpcAttributes;
+import software.amazon.awscdk.services.iam.PolicyDocument;
+import software.amazon.awscdk.services.redshift.Cluster;
+import software.amazon.awscdk.services.redshift.ClusterType;
+import software.amazon.awscdk.services.redshift.Login;
+import software.amazon.awscdk.services.redshift.NodeType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Integration-tests for the DynamoDB connector using the Integration-test module.
+ */
+public class RedshiftIntegTest extends IntegrationTestBase
+{
+    private static final Logger logger = LoggerFactory.getLogger(RedshiftIntegTest.class);
+
+    private static final String REDSHIFT_DB_NAME = "public";
+    private static final String REDSHIFT_DB_PORT = "5439";
+    private static final String REDSHIFT_DB_USERNAME = "integusername";
+    private static final String REDSHIFT_DB_PASSWORD = "IntegPassword1";
+    private static final String REDSHIFT_TABLE_MOVIES = "movies";
+    private static final String REDSHIFT_TABLE_BDAY = "bday";
```

Comment:
these seem like good candidates for the config file... username and password would ideally come from a SecretsManager secret. not be in this code or in a config file.

Reply:
In this instance, we are creating a strictly temporary DB instance/cluster whose persistence is only for the duration of the test, so credentials obfuscation via SecretsManager would be overkill. But, I do agree that we may have other instances in the future (e.g. Vertica connector maybe) where the DB instance might be persistent. So, let's target SecretsManager for a separate PR.

Moved params to config file.